internal/workflow/activities/upload_test.go

```diff
@@ -19,6 +19,7 @@ import (
 // StorageService implements goastorage.Service.
 type StorageService struct {
 	JWTAuthHandler          func(ctx context.Context, token string, scheme *security.JWTScheme) (ctx2 context.Context, err error)
+	ListAipsHandler         func(ctx context.Context, req *goastorage.ListAipsPayload) (res *goastorage.AIPs, err error)
```

Comment:
Wow, maintaining this mock `StorageService` struct seems like a hassle, why not use mockgen to generate a mock from the `goastorage.Service`?

Reply:
I thought the same, and we already have a mock for that service, I'll give it a try ;)

Reply:
It was the client and I needed to update the activity to use the `Client` interface, which also required an update to reflect the new method.